external/Mono.Cecil.overrides

```diff
@@ -8,6 +8,11 @@
     <!-- Cecil sets PublicSign on windows, but we always want to
          PublicSign. -->
     <PublicSign>true</PublicSign>
+    <!-- When we build the cecil submodule, use the same version number
+         as the package dependency. This way the linker ref assembly package
+         will depend on the publicly available package. Note that we can't set
+         just set Version here because Arcade will override it. -->
+    <PackageVersion>$(MonoCecilVersion)</PackageVersion>
```

Comment:
It'd be better to stop the build when the versions don't match but it does not look like it's doable with current Cecil setting